lib/gui/app/scss/components/_button.scss

```diff
@@ -69,6 +69,10 @@ $button-types-styles: (
   danger: (
     bg: $palette-theme-danger-background,
     color: $palette-theme-danger-foreground
+  ),
+  warning: (
+    bg: $palette-theme-warning-background,
+    color: $palette-theme-danger-foreground
```

Comment:
@Shou I'm confused - if you're using `bg: $palette-theme-warning-background`, wouldn't it also make sense to use `color: $palette-theme-warning-foreground` ? Is that what _was_ intended, given that you've added `$palette-theme-warning-foreground: #fff;` to `_theme.scss` ?